cfy_manager/main.py

```diff
@@ -517,20 +517,11 @@ def _print_finish_message():
                 protocol=protocol,
                 ip=manager_config[PUBLIC_IP])
         )
-        print_credentials_to_screen()
-        logger.notice('#' * 50)
-        logger.notice("To install the default plugins bundle run:")
-        logger.notice("'cfy plugins bundle-upload'")
-        logger.notice('#' * 50)
-
-
-def print_credentials_to_screen():
-    password = config[MANAGER][SECURITY][ADMIN_PASSWORD]
-
-    current_level = get_file_handlers_level()
-    set_file_handlers_level(logging.ERROR)
-    logger.notice('Admin password: %s', password)
-    set_file_handlers_level(current_level)
+        print('Admin password: {0}'.format(config[MANAGER][SECURITY][ADMIN_PASSWORD]))
```

Comment:
I think this is the only one that should be a print- the others should go back to being logger.notice shouldn't they?

Reply:
no, there's no reason to put instructions for the user into the logfile, the logfile is for logs instead

Reply:
Fair enough, will approve then.